x-pack/plugins/maps/public/connected_components/map_container/map_container.tsx

```diff
@@ -73,37 +108,32 @@ export class GisMap extends Component {
     }
   };
 
-  _loadGeoFields = async (nextIndexPatternIds) => {
+  _loadGeoFields = async (nextIndexPatternIds: string[]) => {
     if (_.isEqual(nextIndexPatternIds, this._prevIndexPatternIds)) {
       // all ready loaded index pattern ids
       return;
     }
 
     this._prevIndexPatternIds = nextIndexPatternIds;
 
-    const geoFields = [];
-    try {
```

Comment:
Removed try/catch because getIndexPatternsFromIds has try/catch once https://github.com/elastic/kibana/pull/73337 is merged